src/platforms/common/enriching-events/user-feedback.mdx

```diff
@@ -109,7 +109,8 @@ An override for Sentry’s automatic language detection (e.g. `lang=de`)
 | `errorGeneric`   | An unknown error occurred while submitting your report. Please try again.                         |
 | `errorFormEntry` | Some fields were invalid. Please correct the errors and try again.                                |
 | `successMessage` | Your feedback has been sent. Thank you!                                                           |
-| `onLoad`         | n/a                                                                                               |
+| `onLoad`         | n/a - **an optional callback that will be invoked when the widget opens**                         |
+| `onClose`        | n/a - **an optional callback that will be invoked when the widget closes**                        |
```

Comment:
What does the N/A mean here? Like, what is not available about this param?

@AbhiPrasad

Reply:
@shanamatthews my guess was that `n/a` refers to there being no `default value` as per column name. However, the whole table could use some rework given that it's a mixture of default values and descriptions.

Reply:
Ahh, got it, thank you. Yeah, let me take a look at this and see if I can figure out a way to make it more intuitive.